src/main/scala/replpp/ScriptingDriver.scala

```diff
@@ -0,0 +1,51 @@
+package replpp
+
+import dotty.tools.dotc.Driver
+import dotty.tools.dotc.core.Contexts
+import dotty.tools.dotc.core.Contexts.{Context, ctx}
+import dotty.tools.io.{ClassPath, Directory, PlainDirectory}
+import replpp.ScriptingDriver.{MainClassName, MainMethodName}
+
+import java.io.File
+import java.lang.reflect.{Method, Modifier}
+import java.net.URLClassLoader
+import java.nio.file.{Files, Path, Paths}
+import scala.language.unsafeNulls
+
+/** Copied and adapted from dotty.tools.scripting.ScriptingDriver
+ * In other words: if anything in here is unsound, ugly or buggy: chances are that it was like that before... :) */
+class ScriptingDriver(compilerArgs: Array[String], scriptFile: File, scriptArgs: Array[String]) extends Driver {
```

Comment:
can you add a comment about what is different from the original? I've stared at this for a bit, but couldn't spot an actual difference in behavior.

Reply:
yeah that makes sense, will update and merge